packages/config/config/devices/0x031e/vzw32-sn.json

```diff
@@ -1002,6 +1084,26 @@
 			"$import": "~/templates/master_template.json#base_enable_disable_inverted",
 			"label": "Double-Tap Config to Clear Notification",
 			"defaultValue": 0
+		},
+		{
+			"#": "239",
+			"label": "SPI Flash Status",
+			"description": "0 = SPI flash fail; 1 = SPI flash success",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": true
+		},
+		{
+			"#": "240",
+			"label": "Factory Reset",
+			"description": "Set to 120935937 to perform a factory reset",
```

Comment:
This parameter should also have two options: 0 for idle and 120935937 for factory reset.